File: app/src/main/java/me/zhanghai/android/files/filelist/PasteState.java
/*
 * Copyright (c) 2019 Hai Zhang <dreaming.in.code.zh@gmail.com>
 * All Rights Reserved.
 */

package me.zhanghai.android.files.filelist;

import java.util.LinkedHashSet;

import androidx.annotation.NonNull;
import me.zhanghai.android.files.file.FileItem;

public class PasteState {

    public boolean copy;

    @NonNull
    public final LinkedHashSet<FileItem> files = new LinkedHashSet<>();
}

File: app/src/main/java/me/zhanghai/android/files/filelist/PasteState.kt
/*
 * Copyright (c) 2019 Hai Zhang <dreaming.in.code.zh@gmail.com>
 * All Rights Reserved.
 */

package me.zhanghai.android.files.filelist

// TODO: Make immutable?
class PasteState(
    var copy: Boolean = false,
    val files: FileItemSet = fileItemSetOf()
)
